pom.xml

```diff
@@ -218,6 +220,25 @@
             <version>${shrinkwrap.version}</version>
         </dependency>
 
+        <!-- SCM utils -->
+        <dependency>
```

Comment:
Here is the Maven Shared SCM:
https://maven.apache.org/scm/

It lets you run "SCM" command in a generic way, and support more than just Git and SVN.